### DISCUSSION TITLE
React-Table with Two-Way Virtualization or Virtualized Grid

### DISCUSSION THREAD

**Discussion by jasanidarshit** [TanStack/table#2873] [Q&A]
Hey Community
I am rendering a huge data set, not only many rows but many columns as well. There is a good example for vertical virtualization using react-window. React-window also offers "FixedSizeGrid", how can we implement that?

Thanks in advance!

**Comment by acediatic** [comment]
Did you end up working out how to do this @jasanidarshit? I'm facing the same issue at the moment.

**Comment by akash880** [comment]
any solution till now @tannerlinsley
